.yarn/patches/@trezor-connect-web-npm-9.3.0-040ab10d9a.patch

```diff
@@ -1,27 +1,38 @@
 diff --git a/lib/impl/core-in-iframe.js b/lib/impl/core-in-iframe.js
-index c47cf3bff860d6b1855341c00b80fc6c40f9d6d5..275eb0f312ff396819fa406c154a3562842db49d 100644
+index c47cf3bff860d6b1855341c00b80fc6c40f9d6d5..0151bcaac6689ecb26f1b4575ece4f3760ca1b87 100644
 --- a/lib/impl/core-in-iframe.js
 +++ b/lib/impl/core-in-iframe.js
 @@ -116,7 +116,9 @@ class CoreInIframe {
          this._log.enabled = !!this._settings.debug;
          window.addEventListener('message', this.boundHandleMessage);
          window.addEventListener('unload', this.boundDispose);
 -        await iframe.init(this._settings);
-+        const modifiedSettings = Object.assign({}, this.settings);
++        var modifiedSettings = Object.assign({}, this._settings);
 +        modifiedSettings.env = 'webextension';
 +        await iframe.init(modifiedSettings);
          if (this._settings.sharedLogger !== false) {
              iframe.initIframeLogger();
          }
+@@ -132,7 +134,9 @@ class CoreInIframe {
+             }
+             this._popupManager.request();
+             try {
+-                await this.init(this._settings);
++                var modifiedSettings = Object.assign({}, this._settings);
++                modifiedSettings.env = 'webextension';
++                await this.init(modifiedSettings);
```

Comment:
The same settings object is referenced here, so we'd want to use the same modifications. This was present in the original patch